### ISSUE TITLE
The plugin cannot release sbt plugins with maven style

### ISSUE THREAD

**Issue by gzoller** [sbt/sbt-bintray#72]
Apologies if this actually a bintray thing, but I don't know...

I successfully published a package to bintray and was happy about that.  I used the build.sbt below:

```
sbtPlugin := true

scalaVersion in Global := "2.10.5"

version := "0.1"

name := "gitflow-packager"

organization := "co.blocke"

scalacOptions in Compile ++= Seq("-deprecation", "-target:jvm-1.6")

publishMavenStyle := true

addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.5-M1")

bintrayOrganization := Some("blocke")

bintrayReleaseOnPublish in ThisBuild := false

licenses += ("MIT", url("http://opensource.org/licenses/MIT"))

bintrayRepository := "releases"

bintrayPackageLabels := Seq("sbt", "sbt-plugin", "git-flow")
```

The jars were published to:  http://jcenter.bintray.com/co/blocke/gitflow-packager/0.1/<jarfile>

When I tried to use my package (an sbt plugin) with this:  addSbtPlugin("co.blocke" % "gitflow-packager" % "0.1")

I got the following:

```
[warn] ==== jcenter: tried
[warn]   https://jcenter.bintray.com/co/blocke/gitflow-packager_2.10_0.13/0.1/gitflow-packager-0.1.pom
```

As you can see the jcenter path and what sbt is looking for are very different.  What am I missing?

Thanks!

**Comment by gzoller** [comment]
Bump.  Still having issues with this.  The published path in Bintray is different than what sbt is looking for when resolved in a project.  Does sbt-bintray control this?  If so, how?

**Comment by eed3si9n** [comment]
`gitflow-packager_2.10_0.13` is the extra attribute added for sbt plugins to denote the Scala and sbt versions when using Maven layout repositories.

If you have a plugin you should link with sbt community repo - http://www.scala-sbt.org/0.13/docs/Bintray-For-Plugins.html

**Comment by jvican** [comment]
The issue here, @gzoller, is that sbt-bintray cannot release sbt plugins with maven style. This problem has affected me several times in the past, more recently in sbt-release-early. My advice is that you don't use sbt-bintray and use sbt-sonatype instead, and publish directly to Central.
  
Either that, or you can try to fix the URL format that is used for maven style sbt plugins. Someone will need to do it in the end.

**Comment by gzoller** [comment]
Thanks for the reply.  Last night I tried using some setup from sbt-bintray itself, including not publishing maven style.  I got this up to bintray/jcenter and even had an admin link to the sbt/sbt-plugin-releases repo.

Still not resolving tho.  When consuming, sbt looks for ..._2.10_0.13/... path for some reason, instead of separating the dirs.   Is that what you mean by fixing the url format?  How would I do that?

Thanks for any advice!
Greg
